elements/pl-code-output/pl-code-output.py

```diff
@@ -0,0 +1,37 @@
+import prairielearn as pl
+import lxml.html
+import pandas as pd
+
+
+def prepare(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    pl.check_attribs(element, required_attribs=['variable-name'], optional_attribs=['variable-type', 'no-highlight', 'variable-prefix', 'variable-suffix'])
+
+
+def render(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    vartype = pl.get_string_attrib(element, 'variable-type', 'text')
```

Comment:
Please define a global variable for each optional attributes above the `prepare()` function. 

As an example, the`variable-type` attribute would have a global variable of `VARIABLE_TYPE_DEFAULT = 'text'`.